bundles/EcommerceFrameworkBundle/CartManager/Cart/Listing.php

```diff
@@ -23,10 +23,6 @@
  */
 class Listing extends \Pimcore\Model\Listing\AbstractListing
 {
-    /**
-     * @var array
-     */
-    public $carts;
```

Comment:
BC break

Reply:
Yes it is. Perhaps we should leave classes with public item properties as they are. But as most other class variables are protected meanwhile we have to tackle this any time...

Reply:
Resolved